general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -669,11 +702,19 @@ def request() -> requests.Response:
             Returns:
                 The Flask GET request object.
             """
-            return requests.get(
-                f"{self.url}{endpoint}",
-                headers=self.headers,
-                verify=self.verify_https,
-            )
+            if query is None:
```

Comment:
is this necessary? it seems like passing `json=None` should be the same as not passing it